src/hardware/disney.cpp

```diff
@@ -17,63 +17,61 @@
  *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
-#include "dosbox.h"
+#include "disney.h"
 
 #include <cassert>
-#include <queue>
 
-#include "bit_view.h"
 #include "checks.h"
-#include "inout.h"
-#include "mixer.h"
-#include "parallel_port.h"
-#include "pic.h"
-#include "setup.h"
 
 CHECK_NARROWING();
 
-class Disney {
-public:
-	Disney(const io_port_t base_port, const std::string_view filter_pref);
-	~Disney();
+Disney::Disney()
+        : LptDac("DISNEY", ChannelFeature::Mono, initialize_at_mixer_rate)
```

Comment:
`initialize_at_mixer_rate` — nice, but probably better to make this into a generic const and put it into `mixer.h`.